interfaces/builtin/network_control.go

```diff
@@ -92,6 +92,9 @@ network sna,
 /{,usr/}{,s}bin/wpa_passphrase ixr,
 /{,usr/}{,s}bin/wpa_supplicant ixr,
 
+/dev/rfkill rw,
+/dev/rfkill r,
```

Comment:
That's what I thought too. but it does not seem to work like that. There are a few interfaces and also apparmor rules that offer both version of file access rw and r.
Here is the denial I am seeing using network-manager which has /dev/rfkill rw

[   27.192016] audit: type=1400 audit(1468955755.410:22): apparmor="DENIED" operation="open" profile="snap.sw-access-point.hostapd" name="/dev/rfkill" pid=2756 comm="hostapd" requested_mask="r" denied_mask="r" fsuid=0 ouid=0

and this does not happen when I add this change.